devtools/support/debug.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+echo "=====================================HEADER====================================="
+echo "Run this script with your conda env activated"
+echo "Invoke the script like this: "
+echo "./debug.sh | tee -a debug.log"
+echo "Then send us debug.log"
```

Comment:
(not blocking) I found this confusing to read as part of the `echo` output, since by the time I'm reading the output, I've successfully run the script. This may make less experienced users think that they have done something wrong. Putting the instructions in comments might be less ambiguous.

```suggestion
# Run this script with your conda env activated
# Invoke the script like this:
# ./debug.sh | tee -a debug.log
# Then send us debug.log
```

Reply:
I think the idea is that these instructions are at the top of the file. "How to run" instructions will also be duplicated elsewhere